graphql/database/missions/hasSeenCompletedMission.js

```diff
@@ -0,0 +1,32 @@
+import {
+  getPermissionsOverride,
+  MISSIONS_OVERRIDE,
+} from '../../utils/permissions-overrides'
+import UserMissionModel from './UserMissionModel'
+import UserModel from '../users/UserModel'
+
+const override = getPermissionsOverride(MISSIONS_OVERRIDE)
+/**
+ * Records a user's acknowledging a mission notification.
+ *
+ * @param {string} userId - The user's ID.
+ * @param {string} missionId - The mission's ID.
+ * @param {string} action - What action the user took (MISSION_COMPLETE, MISSION_STARTED)
+ */
+const hasSeenCompletedMission = async (userContext, userId, missionId) => {
+  // Get the mission model and modify it.
+  await Promise.all([
+    UserMissionModel.update(override, {
```

Comment:
Don't need an override, right?

Reply:
Same goes for most other places we use `UserMissionModel`:
* updateMissionNotification.js
* getPastUserMissions.js
* getCurrentUserMission.js
* createMission.js

Could you change those, too? Better to use built-in authorizers when we can.

Reply:
updated where we get create or update based on UserId.  Ignored where we fetch all or update all documents based on mission id